ant-node/src/node.rs

```diff
@@ -922,20 +926,26 @@ impl Node {
             });
         }
 
+        let mut peer_scores = vec![];
         while let Some(res) = tasks.join_next().await {
             match res {
                 Ok((peer_id, score)) => {
-                    if score < MIN_ACCEPTABLE_HEALTHY_SCORE {
+                    let is_healthy = score < MIN_ACCEPTABLE_HEALTHY_SCORE;
+                    if !is_healthy {
                         info!("Peer {peer_id:?} failed storage challenge with low score {score}/{MIN_ACCEPTABLE_HEALTHY_SCORE}.");
                         // TODO: shall the challenge failure immediately triggers the node to be removed?
```

Comment:
This sounds like a good idea. Maybe in a next PR so we can test and compare in a testnet to make sure we got the correct threshold?

Reply:
yes